tests/behaviour/csvw-pull.feature

```diff
@@ -0,0 +1,139 @@
+Feature: Testing the CSV-W pull command in the CLI
```

Comment:
For reference, [this](https://github.com/GSS-Cogs/csvcubed-pmd/blob/main/tests/features/csvwcommandline.feature) is the file these behaviour scenarios came from.

Since the `find-where` command isn't being moved over, those scenarios haven't transfered.